docs/source/sysadmin_guide/grafana_configuration.rst

```diff
@@ -0,0 +1,150 @@
+       -e INFLUXDB_USER=lrdata -e INFLUXDB_USER_PASSWORD='test' \
+       --name sawtooth-stats-influxdb influxdb
+
+Grafana
+=======
+
+#. First, build the Grafana Docker image from the Dockerfile included in the sawtooth-core repository.
+
+   .. code-block:: console
+
+     $ cd sawtooth-core/docker
+     $ docker build . -f grafana/sawtooth-stats-grafana -t sawtooth-stats-grafana
+
+#. Run the Grafana container.
+
+   .. code-block:: console
+
+     $ docker run -d -p 3000:3000 --name sawtooth-stats-grafana sawtooth-stats-grafana
+
+#. Next, open the Grafana web page, which should now be accessible by IP address or FQDN at http://{host}:3000.
+
+   .. note::
+
+      The default login credentials are admin:admin.
+
+#. Change the admin password.
+
+   To change the password click the Grafana spiral icon at the top left of the web page and go to the Admin / Profile page, click on "Change Password".
+
+#. Configure Grafana to use InfluxDB as a data source.
+
+   Click on the Grafana spiral icon at the top left of the web page and go to "Data Sources", then click on "Metrics".
+   Change the URL to to the IP or FQDN of the server running InfluxDB. Under "InfluxDB Details", enter the INFLUXDB_USER user and INFLUXDB_USER_PASSWORD password used in the InfluxDB section, then click "Save & Test".
+
+   .. note::
+
+      Sawtooth 1.0.* requires the dashboard from the sawtooth 1-0 branch, while sawtooth 1.1.* can use the dashboard included in the Grafana Docker container from master.
+
+   For Sawtooth 1.0.* only, import the 1.0 dashboard. (Sawtooth 1.1.* can use the dashboard included in the Grafana Docker container.)
+
+   a. Get the 1.0 dashboard from either sawtooth-core/docker/grafana/dashboards/sawtooth_performance.json in the 1-0 branch or download from GitHub directly at `sawtooth_performance.json <https://raw.githubusercontent.com/hyperledger/sawtooth-core/1-0/docker/grafana/dashboards/sawtooth_performance.json>`_.
+
+   b. To import the 1.0 dashboard, click Grafana spiral logo on top left of page and mouse over "Dashboards", click "Import" then click "Upload .json file" and navigate to the path where sawtooth_performance.json was saved. Next select "metrics" in the drop-down menu and click "Import".
+
+Sawtooth Configuration
+======================
+
+Sawtooth metrics are reported by the validator process and the configuration lives in ``/etc/sawtooth/validator.toml``. You can copy the template from ``/etc/sawtooth/validator.toml.example`` if the file doesn't exist yet.
+
+#. Fill in the following values with the configurations used above. ``opentsdb_url`` is the IP / FQDM:port to the InfluxDB instance, ``opentsdb_db`` is the value of INFLUXDB_DB, then fill in the INFLUXDB_USER for ``opentsdb_username`` and INFLUXDB_USER_PASSWORD for ``opentsdb_password`` created above.
+
+   .. code-block:: console
+
+     # The host and port for Open TSDB database used for metrics
+     opentsdb_url = "http://{host}:3000"
```

Comment:
don't use `{host}`

Reply:
there really isn't any address I can assume here, 127.0.0.1 won't work here because this is inside one container without port mapping to the other container.

Reply:
I added a note to replace {host} with the IP or FQDN of the system running the container.